src/components/Tabs/Annotation/LabelEditControls.tsx

```diff
@@ -106,6 +107,40 @@ export default function LabelEditControls(props: PropsWithChildren<LabelEditCont
 
   return (
     <>
+      <label style={{ display: "flex", flexDirection: "row", gap: "6px", marginLeft: "8px" }}>
+        <span style={{ fontSize: theme.font.size.label, width: "max-content" }}>Select by </span>
+        <StyledRadioGroup
+          style={{ display: "flex", flexDirection: "row" }}
+          value={props.selectionMode}
+          onChange={(e) => props.setSelectionMode(e.target.value)}
+        >
+          <TooltipWithSubtitle
+            trigger={["hover", "focus"]}
+            title="Select a single timepoint"
+            subtitle="(Hold Shift to select a range)"
+            placement="top"
+            autoAdjustOverflow={false}
+          >
+            <Radio.Button value={AnnotationSelectionMode.TIME}>Time</Radio.Button>
+          </TooltipWithSubtitle>
+          <Tooltip
+            trigger={["hover", "focus"]}
+            title="Selects range between two timepoints in a track"
+            autoAdjustOverflow={false}
+            placement="top"
+          >
+            <Radio.Button value={AnnotationSelectionMode.RANGE}>Range</Radio.Button>
+          </Tooltip>
+          <TooltipWithSubtitle
+            trigger={["hover", "focus"]}
+            title="Selects entire track"
+            subtitle="(Hold Shift to select a range)"
+            placement="top"
+          >
```

Comment:
Only change in this block is here, where Track also got a tooltip with a subtitle that explains the Shift shortcut key. Everything else was moved directly from below